README.md

```diff
@@ -411,19 +411,26 @@ A number of options are available to allow you to customise the SDK:
 
   This is the identity document capture step. Users will be asked to select the document type and to provide images of their selected document. For driving licence and national ID card types, the user will be able to see and select the supported countries. They will also have a chance to check the quality of the image(s) before confirming.
 
+  If you are migrating from a version older than `6.0.0` please refer to the [`MIGRATION`](MIGRATION.md) documentation for further details about this change.
```

Comment:
Why are we mentioning version `6.0.0` in the `README.md`? I think that integrators should refer to the migration file every time they upgrade to a major version. Also, in order to understand what "this change" means, an integrator would have to look at the commit history.

Reply:
Mentioned that version as it was the last major version. Was trying to be specific about when a major change was introduced. But that's a good point. I'll rephrase it to not refer to a previous version and be clearer what "this change" is.